libflux/flux-core/src/formatter/mod.rs

```diff
@@ -1252,6 +1254,7 @@ impl<'a> Operator<'a> {
                 ast::Operator::NotOperator => 7,
                 ast::Operator::ExistsOperator => 7,
                 ast::Operator::InvalidOperator => 0,
+                ast::Operator::ConditionalOperator => 10,
```

Comment:
Where is this used? I don't see an update to the parser to add the ConditionalOperator to any AST node as a result I don't see any code path where the ConditionalOperator precedence could be queried?

Reply:
That must have been an oversight - thank you for noticing.  I was running different tests to learn how consitionalOperator works in the flux parser/formatter and left that code there.  I will remove this now and will create a fresh PR deleting the nonrelated code. Thanks